agent/consul/server_test.go

```diff
@@ -548,10 +540,7 @@ func TestServer_Expect(t *testing.T) {
 
 	// Wait for the new server to see itself added to the cluster.
 	retry.Run(t, func(r *retry.R) {
-		r.Check(wantPeers(s1, 4))
-		r.Check(wantPeers(s2, 4))
-		r.Check(wantPeers(s3, 4))
-		r.Check(wantPeers(s4, 4))
+		r.Check(wantRaft([]*Server{s1, s2, s3, s4}))
```

Comment:
Are there more places in the tests where it is more correct to call wantRaft instead of wantPeers?

Reply:
I think I got them all but I'll make one more pass through and look over the remaining `wantPeers` uses before I merge. Maybe we can get rid of `wantPeers`.

Reply:
Took a look through the rest and the other uses of `wantPeers` are justified to actually look for voters, given what they are testing.